src/core/InterleavedBufferAttribute.js

```diff
@@ -2,14 +2,16 @@
  * @author benaadams / https://twitter.com/ben_a_adams
  */
 
-THREE.InterleavedBufferAttribute = function ( interleavedBuffer, itemSize, offset ) {
+THREE.InterleavedBufferAttribute = function ( interleavedBuffer, itemSize, offset, normalized ) {
 
 	this.uuid = THREE.Math.generateUUID();
 
 	this.data = interleavedBuffer;
 	this.itemSize = itemSize;
 	this.offset = offset;
 
+	this.normalized = normalized;
```

Comment:
Change to `this.normalized = normalized === true;` to match [BufferAttribute](https://github.com/mrdoob/three.js/blob/master/src/core/BufferAttribute.js#L16)?